From 62971dabc8f56cdff41ce8afa1767c07b4a1f90e Mon Sep 17 00:00:00 2001
From: zhangjun <zhangjun@rock-chips.com>
Date: Fri, 23 Sep 2016 10:12:12 +0800
Subject: [PATCH] arm64: dts: rk3399-sapphire-excavator-edp: disable hdmi audio

due to ff8a0000.i2s can't bound to card "rockchip,hdmi" and
"rockchip,cdn-dp-fb" at the same time

Change-Id: Ie43bf882f0eacb6e87d10ba5eba0fd38dbb5462e
Signed-off-by: zhangjun <zhangjun@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
index e5d0b2d0a98b..e8b91ffe775b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
@@ -68,10 +68,6 @@
 	status = "okay";
 };
 
-&dw_hdmi_audio {
-	status = "okay";
-};
-
 &edp_rk_fb {
 	status = "okay";
 };
-- 
2.35.3

